code.json

```diff
@@ -44,6 +44,51 @@
       "metadataLastUpdated": "2023-01-25"
     }
   },
+  {
+    "name": "knoten",
+    "organization": "U.S. Geological Survey",
+    "description": "GitHub code repository for the knoten package",
+    "version": "0.4.0",
+    "status": "Production",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/knoten/-/raw/0.4.0/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://code.usgs.gov/astrogeology/knoten/-/tree/0.4.0",
+    "downloadURL": "https://code.usgs.gov/astrogeology/knoten/-/archive/0.4.0/knoten-0.4.0.zip",
+    "disclaimerURL": "https://code.usgs.gov/astrogeology/knoten/-/raw/0.4.0/DISCLAIMER.md",
+    "repositoryURL": "https://code.usgs.gov/astrogeology/knoten.git",
+    "vcs": "git",
+
+    "laborHours": 300,
+
+    "tags": [
+      "Planetary",
+      "Remote Sensing",
+      "Photogrammetry",
+      "Testing"
+    ],
+
+    "languages": [
+      "Python"
+    ],
+
+    "contact": {
+      "name": "Adam Paquette",
+      "email": "acpaquette@usgs.gov"
+    },
+
+    "date": {
+      "metadataLastUpdated": "2023-02-27"
```

Comment:
If this was just added, does the date need to be updated?

Reply:
I'm not sure. I am basing these changes from the last bump PR and this date wasn't change then. https://github.com/DOI-USGS/knoten/commit/3c11c814cb21454f2483769dbdc7289b71315db9. Maybe it was a copy and paste error.